### ISSUE TITLE
Tooltip for local IL variables has less information when opcode is in short form

### ISSUE THREAD

**Issue by pardeike** [0xd4d/dnSpy#1537]
The tooltip for a local varable should always show the same information (index + type) regardless if the opcode is in short form or with an index operand.

Currently, short forms like `ldloc.2` only shows this not so very helpful [tooltip](https://imgur.com/a/VZthsmx). Since the prolog with all variables has the type information available, it would be nice that dnSpy shows at least [this](https://imgur.com/a/1ar0EiW) or better that both are consolidated into showing `Variable [Index] of type [type]` or similar.

**Comment by pardeike** [comment]
I just realised that the tooltip is on the operand only. That makes it of course much more complicated to get a tooltip for the short form since there is no operand. Still very inconvenient though.

**Comment by 0xd4d** [comment]
Yeah, the tooltip code only gets some object and in this case it's just an `OpCode` which doesn't have any operand, so it can't show more info.

https://github.com/0xd4d/dnSpy/blob/7241a9f17aed1cac15879102fbac19bee0c1052a/dnSpy/dnSpy/Documents/Tabs/DocViewer/ToolTips/DnlibReferenceDocumentViewerToolTipProvider.cs#L48-L49

The IL disassembler could be updated to instead write something that includes more info, eg. some new class that includes opcode + operand.

**Comment by pardeike** [comment]
Like some custom content after the opcode that holds the index state and has a tooltip showing the type. That would do it.
